src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
The class `.stars` is not following the checklist's guidance on not using tag names for styling. It's good practice to style elements using classes, pseudo-classes, pseudo-elements, and attributes rather than tag names to ensure that the styles are not overly broad and are applied as intended.